connectivity/mbedtls/tools/importer/Makefile

```diff
@@ -60,18 +60,19 @@ rsync:
 	# Copying mbed TLS headers to mbed includes...
 	rm -rf $(TARGET_INC)
 	mkdir -p $(TARGET_INC)
-	mkdir -p $(TARGET_PSA_INC)
 	rsync -a --delete $(MBED_TLS_API) $(TARGET_INC)
-	rsync -a --delete --exclude='crypto_struct.h' $(CRYPTO_API) $(TARGET_PSA_INC)/
+	#
+	# Export common.h into the include path, so PSA Crypto can find it
+	mv $(TARGET_SRC)/common.h $(TARGET_INC)/mbedtls/
```

Comment:
Should this fix be upstreamed to mbedtls?

Reply:
This doesn't apply to the upstream Mbed TLS project, where all non-PSA and PSA source files are in [one directory](https://github.com/ARMmbed/mbedtls/tree/v2.24.0/library).
But in Mbed OS, we split them into non-PSA (`connectivity/mbedtls/source/`) and PSA (`platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_MBED_PSA_SRV/mbedtls/`), as the latter is only used by v7 targets with PSA support. So the path fix is specific to the Mbed OS directory structure.

Reply:
I think it would be a nice improvement though, if the Mbed TLS project can also split non-PSA and PSA sources files in the future (they already do this for headers only).